src/Fixer/DeprecatedFixerInterface.php

```diff
@@ -22,7 +22,7 @@ interface DeprecatedFixerInterface extends FixerInterface
     /**
      * Returns names of fixers to use instead, if any.
      *
-     * @return string[]
+     * @return list<string>
```

Comment:
@keradus FYI: [This can be seen as BC break](https://github.com/kubawerlos/php-cs-fixer-custom-fixers/actions/runs/7352494914/job/20017312577#step:5:50).